UnityProject/Assets/Scripts/UI/Systems/Lobby/Pronouns.cs

```diff
@@ -0,0 +1,7 @@
+public enum PlayerPronoun
```

Comment:
what the difference between this enum and Gender enum?

Reply:
playerpronoun is for well... pronouns
gender is for sprite stuff

Reply:
is body type for Sprite

Reply:
bod you yourself coded in implants being based off of the gender enum